code/src/App.js

```diff
@@ -1,9 +1,191 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import { Classic } from './components/Classic';
+import { Arrabiata } from './components/Arrabiata';
+import { Norma } from './components/Norma';
+import { Scarpariello } from './components/Scarpariello';
+import { Amatriciana } from './components/Amatriciana';
+import { Marinara } from './components/Marinara';
 
 export const App = () => {
+  const [stepCounter, setStepCounter] = useState(1);
+  const [userName, setUserName] = useState('');
+  const [preference, setPreference] = useState('');
+  const [condiment, setCondiment] = useState('');
+  const [texture, setTexture] = useState('');
+  const [sprinkle, setSprinkle] = useState('');
+
+  const handleNextButtonClick = () => {
+    setStepCounter(stepCounter + 1);
+  }
+
+  const handleUserNameInputChange = (event) => {
+    setUserName(event.target.value);
+  }
+
+  const handlePreferenceInputChange = (event) => {
+    setPreference(event.target.value);
+  }
+
+  const handleCondimentChange = (event) => {
+    setCondiment(event.target.value);
+  }
+
+  const handleTextureChange = (event) => {
+    setTexture(event.target.value);
+  }
+
+  const handleSprinkleChange = (event) => {
+    setSprinkle(event.target.value);
+  }
+
+  const renderRecipe = () => {
```

Comment:
The render recipe if/else is very good, it's good that there are statements for different combinations of condiment & texture